README.md

```diff
@@ -596,6 +596,11 @@ Other Style Guides
 
     // good
     const errorMessage = 'This is a super long error that was thrown because of Batman. When you stop to think about how Batman had anything to do with this, you would get nowhere fast.';
+
+    // best
```

Comment:
This isn’t actually best as-is, because it includes all the extra spaces you put in for indentation. You’d need to use https://npmjs.com/dedent , or manually do so in the example, neither of which is better than the one-line example.

Long lines are *just fine*, and our eslint config explicitly disables the max-len check on any line with a string.